src/models/predict_model.py

```diff
@@ -24,6 +26,9 @@
 
 
 def main(args, save_path="", evaluating_test=True):  # noqa: C901
+    device = args.device
```

Comment:
what if we do something like this instead:
```python
device = torch.device("cuda" if torch.cuda.is_available() else "CPU")
```

we should also keep the `import setGPU` but maybe check if `torch.cuda.is_available()` first

Reply:
what if a user has multiple GPUs and he/she/they wants to specify which one to use? Should we give user this option?